docs/csharp/programming-guide/concepts/linq/how-to-count-occurrences-of-a-word-in-a-string-linq.md

```diff
@@ -34,7 +34,7 @@ class CountWords
   
         // Create the query.  Use ToLowerInvariant to match "data" and "Data"
```

Comment:
I like this change @bazile 

Can you update this comment to match the updated code?

Once that's done, I'll :shipit:

```suggestion
        // Create the query.  Use the InvariantCultureIgnoreCase comparision to match "data" and "Data"
```